packages/icons/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "@lmc-eu/spirit-icons",
+  "version": "0.0.0",
+  "description": "Icons of Spirit Design System",
```

I'd keep "for" like in the main README.

```suggestion
  "description": "Icons for Spirit Design System",
```

---

Fixed.